src/Refitter.Core/Settings/ApizrPackages.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+
+namespace Refitter.Core.Settings
+{
+    public enum ApizrPackages
+    {
+        [Description("Install-Package Apizr")]
```

Comment:
Can we use `dotnet add package` instead of `Install-Package` for the Non-Windows